specification/appcomplianceautomation/resource-manager/Microsoft.AppComplianceAutomation/stable/2024-06-27/appcomplianceautomation.json

```diff
@@ -0,0 +1,4625 @@
+      },
+      "readOnly": true
+    },
+    "QuickAssessment": {
+      "type": "object",
+      "description": "A class represent the quick assessment.",
+      "properties": {
+        "resourceId": {
+          "type": "string",
+          "description": "Resource id.",
+          "readOnly": true
+        },
+        "responsibilityId": {
+          "type": "string",
+          "description": "Responsibility id.",
+          "readOnly": true
+        },
+        "timestamp": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The timestamp of resource creation (UTC).",
+          "readOnly": true
+        },
+        "resourceStatus": {
+          "$ref": "#/definitions/ResourceStatus",
+          "description": "Quick assessment status.",
+          "readOnly": true
+        },
+        "displayName": {
+          "type": "string",
+          "description": "Quick assessment display name.",
+          "readOnly": true
+        },
+        "description": {
+          "type": "string",
+          "description": "Quick assessment display name.",
+          "readOnly": true
+        },
+        "remediationLink": {
+          "type": "string",
+          "description": "Link to remediation steps for this quick assessment.",
+          "readOnly": true
+        }
+      }
+    },
+    "Recommendation": {
+      "type": "object",
+      "description": "A class represent the recommendation.",
+      "properties": {
+        "recommendationId": {
+          "type": "string",
+          "description": "The Id of the recommendation.",
+          "readOnly": true
+        },
+        "recommendationShortName": {
+          "type": "string",
+          "description": "The short name of the recommendation. e.g. \"Invalid TLS config\"",
+          "readOnly": true
+        },
+        "recommendationSolutions": {
+          "type": "array",
+          "description": "List of recommendation solutions.",
+          "items": {
+            "$ref": "#/definitions/RecommendationSolution"
+          },
+          "readOnly": true,
+          "x-ms-identifiers": [
+            "recommendationSolutionIndex"
+          ]
+        }
+      }
+    },
+    "RecommendationSolution": {
+      "type": "object",
+      "description": "A class represent the recommendation solution.",
+      "properties": {
+        "recommendationSolutionIndex": {
+          "type": "string",
+          "description": "The index of the recommendation solution.",
+          "readOnly": true
+        },
+        "recommendationSolutionContent": {
+          "type": "string",
+          "description": "The detail steps of the recommendation solution.",
+          "readOnly": true
+        },
+        "isRecommendSolution": {
+          "$ref": "#/definitions/IsRecommendSolution",
+          "description": "Indicates whether this solution is the recommended.",
+          "readOnly": true
+        }
+      }
+    },
+    "ReportBaseProperties": {
```

Comment:
It seems this definition is not referred by any API.

Reply:
Fixed~